docs/content/reference/file-upload.md

```diff
@@ -0,0 +1,129 @@
+---
+title: 'File Upload'
+description: How to upload files.
+linkTitle: File Upload
+menu: { main: { parent: 'reference' } }
+---
+
+Graphql server has an already build it Upload scalar to upload files using a multipart request.
```

Comment:
Thanks for taking the time to write docs, it can be pretty time consuming.

Theres a few small typos in here, having a second read through might be good.

Reply:
Hopefully done.